syntax = "proto2";

package RobotCommandProtobuf;

message KebbiCommand {
  optional string speak_sentence = 1;
  optional int32 yaw = 2;
  optional int32 pitch = 3;
  optional int32 headspeed = 4;
  optional int32 face = 5;
  optional bool hideface = 6;

  enum FaceEnum{
    TTS_AngerA = 0;
    TTS_AngerB = 1;
    TTS_Contempt = 2;
    TTS_Disgust = 3;
    TTS_Fear = 4;
    TTS_JoyA = 5;
    TTS_JoyB = 6;
    TTS_JoyC = 7;
    TTS_PeaceA = 8;
    TTS_PeaceB = 9;
    TTS_PeaceC = 10;
    TTS_SadnessA = 11;
    TTS_SadnessB = 12;
    TTS_Surprise = 13;
    }

  optional int32 stopmove = 7;
  optional int32 motion = 8;
}

